test/goodeggs_money.test.js

```diff
@@ -0,0 +1,351 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable import/no-commonjs */
+const {BigNumber} = require('bignumber.js');
+const {expect} = require('chai');
+
+const Cents = require('../src');
+
+const {fromDollars, isValid, max, min, round, isValidDollars, sum, sumDollars} = Cents;
+
+describe('Cents', function () {
+  describe('constructor', function () {
+    it('can construct with int', function () {
+      const cents = new Cents(1);
+      return expect(cents).to.have.property('value', 1);
+    });
+
+    it('can construct with string', function () {
+      const cents = new Cents('1');
+      return expect(cents).to.have.property('value', 1);
+    });
+
+    it('can construct with BigNumber', function () {
+      const cents = new Cents(new BigNumber(1));
+      return expect(cents).to.have.property('value', 1);
+    });
+
+    it('can construct with Cents', function () {
+      const cents = new Cents(new Cents(1));
+      return expect(cents).to.have.property('value', 1);
```

Comment:
please make sure to remove all these unnecessary `return`s inserted by the decaffeination process!